state/addmachine.go

```diff
@@ -131,9 +136,14 @@ func (st *State) AddMachineInsideMachine(template MachineTemplate, parentId stri
 // AddMachine adds a machine with the given series and jobs.
 // It is deprecated and around for testing purposes only.
 func (st *State) AddMachine(base Base, jobs ...MachineJob) (*Machine, error) {
+	virtualHostKey, err := sshkeys.NewMarshalledED25519()
```

Comment:
This is inconsistent with elsewhere but so long as this is **only** used for tests....

Reply:
By inconsistent with elsewhere, which bit do you mean?